src/stream_wrap.cc

```diff
@@ -46,8 +46,10 @@ using v8::HandleScope;
 using v8::Local;
 using v8::MaybeLocal;
 using v8::Object;
+using v8::PropertyAttribute;
```

Comment:
addtional for quic?

Reply:
No, there's one use of `PropertyAttribute` elsewhere in the file. This was previously being inherited from the stream_base.h headers.